cmd/node/node.go

```diff
@@ -503,7 +503,7 @@ func getEdIdentity() (*signing.EdSigner, error) {
 }
 
 func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
-	log.Info("Starting Spacemesh")
+	log.With().EventInfo("Starting Spacemesh")
```

Comment:
this has no parameters, no need to use event

Reply:
what is the problem with using json here?

Reply:
we need it to be a json since it is an event